### PR TITLE
Making changes suggested by our docs writers to MongoDB and redis sed

### PR DESCRIPTION
**Pull request by fbm3307**
Making changes suggested by our docs writers to MongoDB and redis sed

Signed-off-by: Feny Mehta <fbm3307@gmail.com>

### CODE REVIEW

File: seds/mongodb-sed/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-This helm chart defines a MongoDB Service Endpoint Definition (SED). When the SED is installed it will provide the user with the oportunity to provide connection information as well as credentials to authenticate. The following are the values that can be customized when the SED chart is installed:
+This helm chart defines a MongoDB Service Endpoint Definition (SED). When the SED chart is installed it provides the user with the opportunity to provide binding data as well as credentials to authenticate. The following values are to be customized while installing the SED chart:
```
Comment (Srivaralakshmi) [inline seds/mongodb-sed/README.md]:
```suggestion
This Helm chart defines a MongoDB Service Endpoint Definition (SED). When the SED chart is installed, it provides the user with the opportunity to provide binding data and credentials to authenticate. The following values must be customized while installing the SED chart:
```

Suggestion: The H in Helm is always in capitals. Global comment. Please change in all such instances.

File: seds/mongodb-sed/README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
 1. Hostname
 1. Port
 1. Username
 1. Password
 1. Databasename
 
-The SED Chart will render a secret with the connection information. This secret is compliant with the Service Binding Specification [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by MongoDB SED Chart is a bindable service endpoint that can be projected to workloads using the Service [Binding Direct Secret Reference](https://github.com/servicebinding/spec#well-known-secret-entries).
\ No newline at end of file
+The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding Specification’s [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the MongoDB SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Binding Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```
Comment (Srivaralakshmi) [inline seds/mongodb-sed/README.md]:
```suggestion
The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding specification’s [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the MongoDB SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```
Suggestion: Apply the comment to Redis SED content also.

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@fbm3307 Left some suggestions. PTAL. Thanks!

File: seds/redis-sed/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-This helm chart defines a Redis Service Endpoint Definition (SED). When the SED is installed it will provide the user with the oportunity to provide connection information as well as credentials to authenticate. The following are the values that can be customized when the SED chart is installed:
+This helm chart defines a Redis Service Endpoint Definition (SED). When the SED chart is installed it provides the user with the opportunity to provide binding data as well as credentials to authenticate. The following values are to be customized while installing the SED chart:
```
Comment (Srivaralakshmi) [inline seds/redis-sed/README.md]:
```suggestion
This helm chart defines a Redis Service Endpoint Definition (SED). When the SED chart is installed, it provides the user with the opportunity to provide binding data and credentials to authenticate. The following values must be customized while installing the SED chart:
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@fbm3307 Some more comments. PTAL. Thanks!

File: seds/redis-sed/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
 
 1. Hostname
 2. Password
 
-The SED Chart will render a secret with the connection information. This secret is compliant with the Service Binding Specification [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by MongoDB SED Chart is a bindable service endpoint that can be projected to workloads using the Service [Binding Direct Secret Reference](https://github.com/servicebinding/spec#well-known-secret-entries).
\ No newline at end of file
+The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding Specification’s [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the Redis SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Binding Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```
Comment (Srivaralakshmi) [inline seds/redis-sed/README.md]:
```suggestion
The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding specification’s [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the Redis SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```
```suggestion
The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding Specification’s [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the Redis SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Binding Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```

Comment (Srivaralakshmi) [inline seds/redis-sed/README.md]:
```suggestion
The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding specification’s [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the Redis SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```
```suggestion
The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding Specification’s [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the Redis SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Binding Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@fbm3307 PTAL. Thanks!

**Comment by fbm3307** [comment]
@Srivaralakshmi done with the suggested changes PTAL

**Review by Srivaralakshmi — APPROVED** [review]
@fbm3307 LGTM. Thanks for the changes. Approved!

**Pull request merged**